openbas-front/src/admin/components/payloads/CreatePayload.js

```diff
@@ -38,6 +38,26 @@ class CreatePayload extends Component {
   }
 
   onSubmit(data) {
+    function cleanupObjects(array, idKey, nestedKey, cleanupFn) {
```

Comment:
Buttons at the bottom are reverse

![image](https://github.com/user-attachments/assets/6e67c8df-5bb7-4cba-ac71-b68eeb7b98a3)